go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/kubernetes-csi/external-snapshot-metadata
+
+go 1.22.3
```

Comment:
I can see this changing constantly! We've moved to `1.22.4` while this PR gets reviewed! :-(